tests/Runtimes/RuntimesTest.php

```diff
@@ -194,7 +197,7 @@ public function testExecRuntimes()
             $value = \escapeshellarg((empty($value)) ? 'null' : $value);
             $value = "--env {$key}={$value}";
         });
-
+        Console::log('');
```

Comment:
For debugging?

Reply:
![Bildschirmfoto 2021-04-22 um 17 43 52](https://user-images.githubusercontent.com/1759475/115743818-4e59e900-a392-11eb-981a-b2d2337b5965.png)
Actually to align the checkmarks 😅